src/screen.c

```diff
@@ -47,12 +47,14 @@ void screen_print_debug(const char* message, int duration)
     delay_ms(duration);
 }
 
-void screen_sprintf_debug(int duration, const char* message, ...)
+void screen_sprintf_debug(int duration, const char* fmt, ...)
 {
     va_list args;
-    va_start(args, message);
+    va_start(args, fmt);
     char print[100] = {0};
-    vsnprintf(print, sizeof(print) - 1, message, args);
+    // There is a bug in clang-tidy
+    // See https://bugs.llvm.org/show_bug.cgi?id=41311
+    vsnprintf(print, sizeof(print) - 1, fmt, args); // NOLINT
```

Comment:
>  The  functions  snprintf()  and  vsnprintf()  write  at most size bytes
>       (including the terminating null byte ('\0')) to str.

So actually the `-1` here is not needed?

Reply:
ack!